app/views/submissions/_form.html.erb

```diff
@@ -1,11 +1,19 @@
 <%# locals: (model:) %>
 
-<%= form_with(model:, url:model.new_record? ? submissions_path : submission_path(@model), class: "contents") do |form| %>
+<%= form_with(model:, url: model.new_record? ? submissions_path : submission_path(model), class: "contents") do |form| %>
   <div class="space-y-12">
     <div class="mt-10 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
       <%= form.text_field :name, wrapper: "col-span-full" %>
       <%= form.rich_text_area :description, wrapper: "col-span-full", class: "h-96" %>
 
+
+      <% if model.new_record? %>
```

Comment:
We agreed that the model can be edited once the challenge is open, so this section should also be available for update action.